### DISCUSSION TITLE
All folders with source code are emty

### DISCUSSION THREAD

**Discussion by robkor26** [community/community#122602] [Code Security]
### Select Topic Area

General

### Body

Most of the files on github doesn't contain source code at all. It is just an empty folder named source code. 
How does github allow developers cheating on us?
If developer attach folder sourcecode.zip there should be source code inside it. 
This is so fake. We are being lied to by developers.

**Comment by hafiz-muhammad** [comment]
Looking at your activity on GitHub I'm assuming you're referring to the repository where you made this issue: [https://github.com/Xposed-Modules-Repo/ma.wanam.youtubeadaway/issues/14](https://github.com/Xposed-Modules-Repo/ma.wanam.youtubeadaway/issues/14). 

If you take a look at the last [commit ](https://github.com/Xposed-Modules-Repo/ma.wanam.youtubeadaway/commit/06e20eecda9a48a1aec9511a23179a801657711f) made in that repository, you will see that the module sources have been deleted. 

The repository has a file called '[SOURCE_URL](https://github.com/Xposed-Modules-Repo/ma.wanam.youtubeadaway/blob/master/SOURCE_URL)' that contains a link to this repository: [https://github.com/wanam/YouTubeAdAway](https://github.com/wanam/YouTubeAdAway)
